ale/drivers/__init__.py

```diff
@@ -93,11 +93,21 @@ def load(label, props={}, formatter='ale', verbose=False):
     drivers = chain.from_iterable(inspect.getmembers(dmod, lambda x: inspect.isclass(x) and "_driver" in x.__module__) for dmod in __driver_modules__)
     drivers = sort_drivers([d[1] for d in drivers])
 
+    try:
+        # Try default grammar for pds3 label
+        parsed_label = parse_label(label)
+    except ValueError:
+        # If pds3 label fails, try isis grammar
+        parsed_label = parse_label(label, pvl.grammar.ISISGrammar)
+    except:
+        # If both fail, raise the error to the calling function
+        raise
```

Comment:
If both fail, then  this should just pass the file and not a parsed label. This way if we implement something like PDS4 XML label support then it doesn't stop at this raise and can just be inefficient until we add xml label parsing.